I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/pride.rpy", line 839, in script
    $ studyday +=1
  File "game/pride.rpy", line 839, in <module>
    $ studyday +=1
NameError: name 'studyday' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/pride.rpy", line 839, in script
    $ studyday +=1
  File "C:\Users\emmyi\Downloads\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\emmyi\Downloads\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/pride.rpy", line 839, in <module>
    $ studyday +=1
NameError: name 'studyday' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
CU_HEdoubleL 1.0
Sun May 17 15:35:49 2020
